modules/mobile/app/controllers/mobile/v0/claims_and_appeals_controller.rb

```diff
@@ -76,7 +83,7 @@ def upload_documents
         )
         raise Common::Exceptions::ValidationErrors, document_data unless document_data.valid?
 
-        jid = document_upload_service.upload_document(document_data)
+        jid = evss_claim_service.upload_document(document_data)
         render json: { data: { job_id: jid } }, status: :accepted
```

Comment:
do we want to add 'happy logging' here for the job id? Then we could look up the job status in sidekiq if it ever needed debugging.

Reply:
I added a sentry log to both doc upload and request decision reporting the claim id and the jid for that request.